.github/workflows/deploy-application.yml

```diff
@@ -118,3 +118,18 @@ jobs:
           cf_org: gsa-tts-oros-fac
           cf_space: ${{ env.space }}
           command: cf run-task gsa-fac -k 6G -m 1G --name load_data --command "./load_data.sh"
+
+      # This has to happen after an application deployment because the manifest (currently) is responsible
+      # for binding the "logdrain service" to the "gsa-fac application". This also needs to be done
+      # based on the suspicion that fluentbit cannot register the incoming logs when it is initially
```

Comment:
As far as I can tell, yes, this is reliably addressing the issue. Once everything is configured and the app is deploying, I only ever saw it go to a 201 instead of the 502 after a restart, or something needed to minorly modify the module. Probably something to investigate, but, I suppose that is the beauty of being the first consumers of this 🤣